worker/uniter/remotestate/watcher.go

```diff
@@ -204,9 +204,8 @@ func (w *RemoteStateWatcher) CommandCompleted(completed string) {
 	}
 }
 
-func (w *RemoteStateWatcher) setUp(unitTag names.UnitTag) error {
+func (w *RemoteStateWatcher) setUp(unitTag names.UnitTag) (err error) {
```

Comment:
don't need this

Reply:
just removed the `var err error` below, so we need this